packages/rancher-gatekeeper/package.yaml

```diff
@@ -1,9 +1,9 @@
-url: https://open-policy-agent.github.io/gatekeeper/charts/gatekeeper-3.3.0.tgz
-packageVersion: 1
+url: https://open-policy-agent.github.io/gatekeeper/charts/gatekeeper-3.7.0.tgz
+packageVersion: 0
 additionalCharts:
   - workingDir: charts-crd
     crdOptions:
       templateDirectory: crd-template
       crdDirectory: crd-manifest
       addCRDValidationToMainChart: true
-doNotRelease: true
\ No newline at end of file
+doNotRelease: false
```

Comment:
Can we just remove this field instead of setting it to false?

Reply:
Rmoved the doNotRelease field